test/Inventory.t.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Test, console2} from "../lib/forge-std/src/Test.sol";
+import {IInventory} from "../contracts/interfaces/IInventory.sol";
+import {IDiamondCut} from "../contracts/interfaces/IDiamondCut.sol";
+import {IDiamondLoupe} from "../contracts/interfaces/IDiamondLoupe.sol";
+import {Inventory} from "../contracts/Inventory.sol";
+import {Diamond} from "../contracts/Diamond.sol";
+import {DiamondCutFacet} from "../contracts/facets/DiamondCutFacet.sol";
+import {OwnershipFacet} from "../contracts/facets/OwnershipFacet.sol";
+import {DiamondLoupeFacet} from "../contracts/facets/DiamondLoupeFacet.sol";
+import {InventoryFacet} from "../contracts/facets/InventoryFacet.sol";
+import {MockERC721} from "../contracts/mocks/MockErc721.sol";
+
+import {IERC721} from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+contract InventoryTests is Test {
+    IInventory public inventory;
+    IERC721 public nftCharacters;
+
+    address owner = address(0x1);
```

Comment:
address alice = makeAddr("aliceOwner");
```suggestion
    address owner = makeAddr("owner");

```

Reply:
might make carrying around or figuring out whose who easier whenever you need it just a util makes address making easier

Reply:
Done.